### PR TITLE
BREAKING CHANGE: Add spirit infix to data attr names for components and JS plugins #DS-585

### PR DESCRIPTION
**Pull request by crishpeen**
## Description

Add `spirit` to all data-api attributes of our components.

### Additional context

See each commit for a list of affected components.

### Issue reference

#DS-585

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md).
- [x] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Comment by literat** [comment]
@crishpeen Please recheck your web-react commit, because `data-placement` is used by FloatingUI to align the arrow of the tip (https://floating-ui.com/docs/arrow#placement). Thus this should be without infix.

File: packages/web-react/src/components/Tooltip/README.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ const { getReferenceProps, getFloatingProps } = useInteractions([hover, focus, d
       top: y ?? 0,
       left: x ?? 0,
     }}
-    data-placement={placement}
+    data-spirit-placement={placement}
```
Comment (literat) [inline packages/web-react/src/components/Tooltip/README.md:158]: I think this is part of Floating UI. Must check!

File: packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidth.twig
Comment (literat) [inline packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidth.twig:1]: New empty file added?

File: packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidthMobile.twig
Comment (literat) [inline packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidthMobile.twig:1]: New empty file added?

File: packages/web/src/js/utils/ComponentFunctions.ts [hunk 2]
```diff
@@ -3,10 +3,10 @@ import EventHandler from '../dom/EventHandler';
 import SelectorEngine from '../dom/SelectorEngine';
 import { getElement, getTriggerOrTarget, Aim } from './Elements';
 
-type DataTriggerAttribute = 'data-toggle' | 'data-dismiss';
+type DataTriggerAttribute = 'data-spirit-toggle' | 'data-spirit-dismiss';
 
-const ATTRIBUTE_DATA_TOGGLE = `data-toggle`;
-const ATTRIBUTE_DATA_DISMISS = `data-dismiss`;
+const ATTRIBUTE_DATA_SPIRIT_TOGGLE = `data-spirit-toggle`;
```
Comment (literat) [inline packages/web/src/js/utils/ComponentFunctions.ts:8]: No need to add infix to constants, imho.

File: packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidth.twig
Comment (literat) [reply packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidth.twig:1]: Fixed.

File: packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidthMobile.twig
Comment (literat) [reply packages/web-twig/src/Resources/components/Dropdown/stories/DropdownFullwidthMobile.twig:1]: Fixed.

File: packages/web/src/js/utils/ComponentFunctions.ts (hunk 2 above)
Comment (literat) [reply packages/web/src/js/utils/ComponentFunctions.ts]: Fixed.

File: packages/web-react/src/components/Tooltip/README.md (hunk 1 above)
Comment (literat) [reply packages/web-react/src/components/Tooltip/README.md:158]: My bad, it is a reference to SCSS - https://github.com/lmc-eu/spirit-design-system/pull/935/files#diff-d188d47603a8e0ae11d30058f82a2f99f6d66697bab530888ffdcc52d706cb63

**Review by literat — APPROVED** [review]
LGTM 👍

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**